### PR TITLE
Updates to support app-builder new form, updates pom and create a doc…

### PR DESCRIPTION
**Pull request by Kerruba**
Basically in this I've updated the pom to generate the container with the jib-pluing,
The entity for an DigitalExchange now uses clientId and clientSecret for consistency with keycloak and all the other proejcts.
I had to update how the resources are read from the zip file as the one generated from git didn't include directories

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: docker-compose.yml [hunk 1]
```diff
@@ -11,3 +11,18 @@ services:
...
+    ports:
+      - "9999:8084"
+    environment:
+      PORT: 8084
+      DB_HOST: database
+      KEYCLOAK_AUTH_URL: "http://test-keycloak.192.168.1.9.nip.io/auth"
+      KEYCLOAK_CLIENT_ID: "test-entando-de"
+      KEYCLOAK_CLIENT_SECRET: "e8b31c02-2021-43d4-9c44-e261ac61a8cf"
+      ENTANDO_URL: "http://test-entando.192.168.1.9.nip.io/entando-de-app"
+      KUBERNETES_ENTANDO_APP_NAME: "test-entando"
```
Comment (ampie) [inline docker-compose.yml]: Could we perhaps changes this just to ENTANDO_APP_NAME? We have a similar variable for plugins and we just call it ENTANDO_PLUGIN_NAME. Let's loose the 'KUBERNETES_' prefix.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by Kerruba** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**